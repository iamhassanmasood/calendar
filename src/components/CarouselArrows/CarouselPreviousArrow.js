import React from "react";
import { LeftOutlined } from "@ant-design/icons";

const CarouselPreviousArrow = (props) => {
  const { className, style, onClick } = props;
  return (
    <LeftOutlined
      className={`${className} arrows-class`}
      style={{ ...style }}
      onClick={onClick}
    />
  );
};

export default CarouselPreviousArrow;
