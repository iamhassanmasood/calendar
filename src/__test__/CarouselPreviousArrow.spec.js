import React from "react";
import { shallow } from "enzyme/build";
import CarouselPreviousArrow from "./../components/CarouselArrows/CarouselPreviousArrow";

describe("check CrouselArrpw rendering on dom", () => {
  it("mounts without crashing", () => {
    const wrapper = shallow(<CarouselPreviousArrow />);
    wrapper.unmount();
  });
  it("check dom tree rendering", () => {
    let tree = shallow(<CarouselPreviousArrow />);
    expect(tree).toMatchSnapshot();
  });
});
