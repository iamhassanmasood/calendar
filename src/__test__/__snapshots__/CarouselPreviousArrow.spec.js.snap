// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check CrouselArrpw rendering on dom check dom tree rendering 1`] = `ShallowWrapper {}`;
